### PR TITLE
Riam Alali-week2hw

### PR DESCRIPTION
**Pull request by RiamAlali**

### CODE REVIEW

File: week2homework.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+console.log (TurkishGreeting)
+let DutchGreeting = "Hallo Wereld!";
+console.log (DutchGreeting)
+
+//2
+console.log ('I\'m awesome');
+
+//3
+let x ;
+console.log ("the value of my variable x will be:EvenNumber");
```
Comment (marcianoviereck) [inline week2homework.js:18]: Are you sure that the value of `x` is `EvenNUmber` ?
(I don't think it will be that value)

File: week2homework.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+console.log (x);
+x = 8 ;
+console.log ("the value of my variable x will be:EvenNumber");
```
Comment (marcianoviereck) [inline week2homework.js:21]: Are you sure that the value of x is `EvenNumber`?

File: week2homework.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+console.log (x)
+
+//4
+let y = "HackYourFuture"
+console.log ("the value of the string is "+"HackYourFuture")
```
Comment (marcianoviereck) [inline week2homework.js:26]: Please don't forget to add `;` at the end of the line:)

File: week2homework.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+console.log (y)
+
+//5
+let z = 7.25;
+console.log(z);
+let a = (Math.round(z));
+console.log(a);
+
+let highest;
+if (z>a){
```
Comment (marcianoviereck) [inline week2homework.js:39]: Good! You can also use [math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max), but this solution is fine:)

File: week2homework.js [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    highest=z;
+}
+else{
+    highest=a;
+}
+console.log(highest);
+
+
+//6
+let FavoriteAnimals = [ ];   //6.1
```
Comment (marcianoviereck) [inline week2homework.js:49]: You should not use capital letters for the variable names.

So `FavoriteAnimals` should be `favoriteAnimals`.

See [The naming conventions](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md)

File: week2homework.js [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+console.log("the value of my array is : Empty."); //6.2
+console.log(FavoriteAnimals); //6.3
+let MyFavoriteAnimals = [ 'Deer' , 'Horse' , 'Koala' ]; //6.4
+console.log (MyFavoriteAnimals);  //6.5
+let NewMyFavoriteAnimals = MyFavoriteAnimals.concat(['baby pig']); //6.6
```
Comment (marcianoviereck) [inline week2homework.js:54]: Ah nice, you can also use 
`MyFavoriteAnimals.push('baby pig');`

Which might be an easier solution

File: week2homework.js [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+let type3 = " Hello JavaScript " ;
+console.log ("The value of my variable type3 is: " + type3);
+console.log (typeof(type3));
+let type4 = "Class21" ;
+console.log ("The value of my variable type4 is: " + type4);
+console.log (typeof(type4));
+
+//8.5
+if (typeof(type1)===typeof(type2))
+{console.log('type1 and type2 are SameType');}
```
Comment (marcianoviereck) [inline week2homework.js:78]: Tip: The code is easier to read if you add more spaces in between the if statement, like this:

```
if (typeof(type1)===typeof(type3)) {
  console.log('type1 and type3 are SameType');
} else {
  console.log('type1 and type3 are NOT SameType');
}
```

File: week2homework.js [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+if (typeof(type3)===typeof(type4))
+{console.log('type3 and type4 are SameType');}
+else {console.log('type3 and type4 are NOT SameType');}
+
+
+
+
+
+//9
+let X = 7 ;
```
Comment (marcianoviereck) [inline week2homework.js:106]: please use lowercase `x`

File: week2homework.js [hunk 9]
```diff
@@ -0,0 +1,121 @@
+//1
```
Comment (marcianoviereck) [inline week2homework.js:1]: I see that you did all your homework in one file, but the assignment was to make one file for each part of the homework:

> For all the following exercises create a new .js file. Try to find a proper name for each file or make a small comment about what it does inside for future reference.

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
Overall it looks good! Just some smaller style remarks and could you split your homework up in separate files?

File: week2homework.js (hunk 9 above)
Comment (RiamAlali) [reply week2homework.js:1]: Thank you for your review, I'v made changes as advised, could you please re-check?

**Comment by wouterkleijn** [comment]
Dear @RiamAlali Thanks a lot for addressing Marciano's feedback - very professional indeed! @marcianoviereck Can you check if there is anything left to be done?

**Comment by marcianoviereck** [comment]
@wouterkleijn Yes the homework for week 2 is now approved, but I still need to check the homework for week 3..

File: week2homework.js (hunk 5 above)
Comment (marcianoviereck) [reply week2homework.js:49]: Can you fix this one?

File: week2homework.js (hunk 8 above)
Comment (marcianoviereck) [reply week2homework.js:106]: And this one, then the homework is approved!

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
Some very minor changes left, over all its good just two small things!

**Pull request closed without merging**